docs/configuring-playbook-collabora.md

```diff
@@ -0,0 +1,135 @@
+# Setting up Collabora integration
+
+**WARNING**: Work in progress - Currently only external reverse proxy supported.
+
+The following configuration enables you to install the [Collabora Online Development Edition (CODE)](https://www.collaboraoffice.com/) document server.
+You will have a fully functioning [WOIP server]() that (together with the [Nextcloud Office App](https://apps.nextcloud.com/apps/richdocuments)) enables you to
```

Comment:
WOIP -> WOPI? Where should this link? Maybe [here](https://learn.microsoft.com/en-us/openspecs/office_protocols/ms-wopi/0f0bf842-6353-49ed-91c0-c9d672f21200)?

Reply:
Yes, I forgot to add that!